percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -309,6 +310,10 @@ install_deps() {
 #
       install_golang
       install_gcc_54_centos
+      if [ x"$RHEL" = x6 ]; then
+          mv /usr/bin/as /usr/bin/as_back
+          ln -s /opt/percona-devtoolset/root/usr/bin/as /usr/bin/as
```

Comment:
Why do we need to back up this file? Maybe we can just force the creation of the link:
ln -sf /opt/percona-devtoolset/root/usr/bin/as /usr/bin/as

Reply:
there are several points:
1 the node can be picked up by jenkins for next build and in this case it would be inconsistent
2 this script can be used by users to build they own copies of psmdb so we shouldn't break their system